src/main/com/homework/week12/stopwatch/DisplayType.java

```diff
@@ -0,0 +1,18 @@
+package com.homework.week12.stopwatch;
+
+public enum DisplayType {
+    SECONDS_AND_MILLIS {
+        @Override
+        public TimeFormatStrategy timeStrategy() {
+            return new MilisecondsAndSecondFormat();
```

Comment:
you could inject via constructor

Reply:
Sorry. not sure if i understand. what i need inject via constructor

Reply:
every time when timeStrategy() invoked you create a new object. whereas you could return the same object every time. I Suggest to create TimeFormatStrategy  final field instead